views/index.pug

```diff
@@ -1,21 +1,68 @@
 html
-  include partials/head
+head
+    link(rel="stylesheet" href="/css/styles.css")
 
-  body
-    h1= schittscreek.name
+    body
+     h1.title Welcome to Schitt's Creek!
+      h2 Select a Method:
+      li.methodsInfo 
+        p 
+          ul.methods GET/allCharacters
+        div#getAll
+            p Method: GET 
+            p Route: http://localhost:4053/characters/allCharacters 
+            p Headers:   
+            p Description: Returns a list of all of the characters in Schitt's Creek.
+      li.methodsInfo 
+        p 
+          ul.methods GET/characterByName
+        div 
+            p Method: GET 
+            p Route: http://localhost:4053/characters/characterByName 
+            p Headers:   
+            p Description: Returns the character with the name provided in the route.  
+      li.methodsInfo 
+        p
+          ul.methods GET/characterByQuote
+        div 
+            p Method: GET 
+            p Route: http://localhost:4053/characters/characterByQuote
+            p Headers:   
+            p Description: Returns the character associated with the provided quote. 
+      li.methodsInfo 
+        p 
+          ul.methods GET/characterByOccupation
+        div 
+            p Method: GET 
+            p Route: http://localhost:4053/characters/characterByOccupation
```

Comment:
All these `characterBy` routes feel weird. I'd expect either `/characters/roland` or `/characters/mayor` to work in the same way as we did with directors and titles in the Movie API using the same route/controller. If you want to segregate them you could do `/characters/occupation/mayor` that is must more REST like.